### ISSUE TITLE
Event Request - To all CreateDim Functions

### ISSUE THREAD

**Issue by MiRo65** [Microsoft/ALAppExtensions#295]
Please add new Event in all CreateDim functions (in tables for example Table 37) as last line, in Addition to the existing Event OnAfterCreateDimTableIds, to add function after the Dims are created.
Thank you.

![image](https://user-images.githubusercontent.com/36627515/45615868-0c47c880-ba6e-11e8-91af-13055a41a679.png)

**Comment by JesperSchulz** [comment]
Closing as result of conversation. Resolution appears to be "Won't fix". Please submit alternative issues if you still are blocked on this.
